Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+pipeline {
+    agent {
+        dockerfile { filename 'Dockerfile.ci' }
+    }
+
+    stages {
+        stage('Dependencies') {
+
+            steps {
+                sh 'mkdir ~/.npm-global'
```

Comment:
all global installations should be part of `Dockerfile.ci`. Look here for example: https://github.com/bringg/customer-tracker/blob/master/Dockerfile.ci